tests/save_load.test.js

```diff
@@ -153,3 +170,25 @@ test('load data from a file', async () => {
 
   expect(loadedLogs).toEqual(file);
 });
+
+test('handling error when promise rejected from reading file', async () => {
+  const readAsText = jest.spyOn(FileReader.prototype, 'readAsText');
+  const readFileFn = jest.spyOn(load, 'readFile');
+
+  const file = JSON.stringify([{ prop: 'log' }]);
+  const blob = new Blob([file], {
+    type: 'application/json',
+  });
+
+  readFileFn.mockImplementation(() => {
+    return Promise.reject('loaderror');
+  });
+
+  const loadedLogs = await load.loadLogsStr(blob).catch((error) => {
```

Comment:
Use `expect(...).rejects` instead of manual `.catch`: https://jestjs.io/docs/en/expect.html#rejects

There are two issues with the current `.catch` + `expect`:
- It's lots of boilerplate and any resulting logged assertion/expectations are not as obvious.
- If the method doesn't reject, the test passes (when it should have failed instead!)